Note.cpp

```diff
@@ -0,0 +1,168 @@
+/******************************************************************
+    Note.cpp
+
+    implementation of the Note structure
+******************************************************************/
+
+#include "Note.h"
+
+Note::Note()
+{
+    stringNum = 1;
+    fretNum = 0;
+    finger = none;
+    //std::cout << "Note default CTOR" << std::endl; 
+}
+
+Note::Note(int str, int fret, Finger digit, char note)
+{
+    stringNum = str;
+    fretNum = fret;
+    finger = digit;
+
+    key = note;
+
+    //std::cout << "Note CTOR" << std::endl;
+}
```

Comment:
Your default constructor should call the constructor that takes parameters, to initialize the members.
You should initialize your members in the member initializer list.
- [ ] revise

Reply:
fixed